### PR TITLE
Translated some more pages

### PR DESCRIPTION
**Pull request by D-Maxwell**
I'm back ! There's been an expected amount of progress since I had to leave. Glad to see things are moving.

### CODE REVIEW

File: docs/vso/manpage.fr.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+## CONSULTER EGALEMENT
+
+- [`apx`](/docs/apx)
+- [`abroot`](/docs/vso)
+
+## AUTEUR
+
+```text
+Les contributeurs de Vanilla OS
```
Comment (kbdharun) [inline docs/vso/manpage.fr.md:193]: Can you add your name too here along with Contributors of Vanilla OS? 

Other than that this LGTM

**Review by kbdharun — APPROVED** [review]

**Comment by D-Maxwell** [comment]
Open for the standardisation of the vocabulary used, as well as some arbitrary choices like should we translate `manpage` and the titles of the tables at the beginning of most pages

File: docs/vso/manpage.fr.md (hunk 1 above)
Comment (D-Maxwell) [reply docs/vso/manpage.fr.md:193]: Alright done

Comment (D-Maxwell) [reply docs/vso/manpage.fr.md]: As explained below I believe further revisions are to be made before merging this PR.

**Comment by D-Maxwell** [comment]
Because I believe these tables's contents are to be retrieved through a terminal by their title, thus why I have avoided translating the headers (`title` and `description`).

**Pull request merged**